hack/lib/golang.sh

```diff
@@ -321,7 +321,7 @@ EOF
   local go_version
   IFS=" " read -ra go_version <<< "$(go version)"
   local minimum_go_version
-  minimum_go_version=go1.10.1
+  minimum_go_version=go1.10.2
```

Comment:
This will likely fail because all the test-infra hasn't been upgraded yet. I don't think we need to enforce this as a minimum quiet yet either.

Reply:
Makes sense, removed.

Reply:
@BenTheElder can we bump test-infra?

Reply:
though in general I think the minimum go version here is more important for minor version bumps (e.g. 1.9 -> 1.10) than patch version bumps.

Reply:
yes, though not much picks up the go versions there, since mostly we either pick up the (in CI doubly so) containerized or bazel builds.

Reply:
discussed offline, we'll bump all the things

Reply:
I've re-added the bump here!